src-docs/src/views/home/home_view.js

```diff
@@ -39,7 +39,9 @@ export const HomeView = () => (
           <EuiFlexItem grow={false}>
             <p>
               Version:{' '}
-              <Link to="/package/changelog">
+              <Link
+                aria-label={`Version ${pkg.version}, View on GitHub`}
```

Comment:
This link does not go to Github, it goes to the rendered changelog page.
```suggestion
                aria-label={`Version ${pkg.version}, View changelog`}
```